docs/source/tracking.rst

```diff
@@ -133,6 +133,29 @@ an optional ``artifact_path``.
 logged to.
 
 
+Automatic logging
```

Comment:
```suggestion
Enable Automatic Logging from TensorFlow (experimental)
```

Just so it's super clear what this is from the header / in the table of contents.